python/fastquant/strategies/base.py

```diff
@@ -30,14 +33,60 @@
     SELL_PROP,
     SHORT_MAX,
 )
+from fastquant.utils.strategy_logging import StrategyLogger
 
 
 class BaseStrategy(bt.Strategy):
     """
     Base Strategy template for all strategies to be added to fastquant
+
+    Attributes
+    ----------
+    init_cash : float
+        inital cash
+    buy_prop : float
+        buy proportion, percentage of available units to buy, setting to 1.0 buys all afforded units
+    sell_prop : float
+        sell proportion, percentage of owned units to sell
+    commission: float
+        Percentage of comission
+    stop_loss : float
+        stop loss percent
+    stop_trail : float
+        stop trail percent
+    execution_type : str {`market`,`close`} (default=`market`)
+        If set to `market`, excecutes the action (buy or sell) at current bar (or day),
+        If set to `close`, executees the action on the next available bar
+    allow_short : bool (default=False)
+        allow short selling
+    short_max : float
+        The max incremental short allowed is the short that would lead to a cumulative short position
+        equal to the maximum short position (initial cash times the maximum short ratio, which is 1.5 by default)
+    add_cash_amount : int (default=0)
+        Added cash amount for periodic cash-ins. (can be used in cost averaging)
+    add_cash_frequency : str {`W` or `M`} (default=`W`)
```

Comment:
```suggestion
    add_cash_frequency : str {`W`, `M`, or custom cron notion} (default=`W`)
```

Reply:
Just added clarification that any custom cron notation can be used to indicate cash inflow frequency / timing